src/server.ts

```diff
@@ -3,5 +3,5 @@ import app from './app'
 const PORT = process.env.PORT || 3000
 
 app.listen(PORT, () => {
-    console.log(`Magic Happens on port ${PORT}`)
+    global.ctx.log.info(`Magic Happens on port ${PORT}`)
```

Comment:
Add console.log also.

Reply:
Why console.log also ?

Reply:
to check log while developing on development.